client/interledger/js/components/interledger.js

```diff
@@ -2,111 +2,48 @@ import React from 'react';
 
 import { Navbar } from 'react-bootstrap/lib';
 
-import { safeMerge } from 'js-utility-belt/es6';
-
 import AccountList from '../../../lib/js/react/components/accounts';
+import AccountDetail from './account_detail';
 
 import AssetActions from '../../../lib/js/react/actions/asset_actions';
-import AssetStore from '../../../lib/js/react/stores/asset_store';
-
-const Interledger = React.createClass({
-
-    getInitialState() {
-        const assetStore = AssetStore.getState();
 
-        return safeMerge(
-            {
-                activeAccount: null,
-                activeAsset: null,
-                activeLedger: null
-            },
-            assetStore
-        );
-    },
+import BigchainDBMixin from '../../../lib/js/react/mixins/bigchaindb_mixin';
```

Comment:
![](http://memesvault.com/wp-content/uploads/Nooo-Meme-Darth-Vader-04.jpg)

Could we use composition here somehow?
If now, I think @SohKai has us covered anyways with his `SpecExtender`, no?